kitsune/settings.py

```diff
@@ -442,8 +442,6 @@ def immutable_file_test(path, url):
             "undefined": "jinja2.Undefined",
             "extensions": [
                 "waffle.jinja.WaffleExtension",
-                "jinja2.ext.autoescape",
-                "jinja2.ext.with_",
```

Comment:
These are both built into the core of `jinja2` now. They're no longer available from `jinja2.ext`.

Reply:
Gotcha.